scripts/build.py

```diff
@@ -156,29 +226,99 @@ def process_js(source_path, target_path):
     _minify(source_path, target_path)
 
 
-def build_minified_third_party_libs(output_directory):
+def get_dependency_filepaths(dependency, filepaths):
+    """Extracts dependency filepaths into filepaths object.
+
+    Args:
+        source_paths: dict(str, str). Dependency information object.
+        filepaths: dict(str, list(str)). Filepaths object with three lists each
+            for different file type (js, css, fonts).
+    """
+    if 'targetDir' in dependency:
+        dependency_dir = dependency['targetDir']
+    else:
+        dependency_dir = (
+            dependency['targetDirPrefix'] + dependency['version'])
+    if 'bundle' in dependency:
+        dependency_bundle = dependency['bundle']
+        for css_file in dependency_bundle.get('css', []):
+            filepaths['css'].append(
+                os.path.join(
+                    THIRD_PARTY_STATIC_DIR, dependency_dir, css_file))
+        for js_file in dependency_bundle.get('js', []):
+            filepaths['js'].append(
+                os.path.join(
+                    THIRD_PARTY_STATIC_DIR, dependency_dir, js_file))
+        if 'fontsPath' in dependency_bundle:
+            for extension in FONT_EXTENSIONS:
+                filepaths['fonts'].append(
+                    os.path.join(
+                        THIRD_PARTY_STATIC_DIR, dependency_dir,
+                        dependency_bundle['fontsPath'], extension))
+
+
+def get_dependencies_filepaths():
+    """Extracts dependencies filepaths from manifes.json file into dictionary
+
+    Returns:
+        dict(str, list(str)). Filepaths object with three lists each for
+        different file type (js, css, fonts).
+    """
+    filepaths = {
+        'css': [],
+        'js': [],
+        'fonts': []
+    }
+    with open(MANIFEST_FILE_PATH, 'r') as json_file:
+        manifest = json.loads(
+            json_file.read(), object_pairs_hook=collections.OrderedDict)
+    frontend_dependencies = manifest['dependencies']['frontend']
+    for dependency in frontend_dependencies.values():
+        get_dependency_filepaths(dependency, filepaths)
```

Comment:
Actually, why do you need to pass in filepaths at all? Probably cleaner for get_* methods to actually return a thing rather than modify something in-place. (If you go with this, don't forget to add a Returns docstring too.)

Reply:
Done.